pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -697,19 +696,19 @@ public class Utils {
             }
         }
         for (int x = beginLine; x <= endLine; x++) {
-            String line = sourceUnit.source.getLine(x, null);
+            String line = sourceUnit.source.getLine(x, null)
             if (line == null)
-                throw new AssertionError("Unable to get source line"+x);
+                throw new AssertionError("Unable to get source line"+x)
 
             if (x == endLine) {
-                line = line.substring(0, endLineLastColumn - 1);
+                line = line.substring(0, endLineLastColumn - 1)
```

Comment:
Could potentially be cleaned a bit more using `str.take` and `str.drop`. Depending on your definition of cleaning :)

Reply:
I have never actually used those yet, so I'll stick with what I'm familiarish with. =)